docker-compose.yml

```diff
@@ -5,6 +5,19 @@ volumes:
     etcher_gnupg:
     etcher_pki:
 services:
+  fan-control:
+      image: bh.cr/balena/fan_control_arm
+      privileged: true
+      restart: always
```

Comment:
causes crash-looping on current OS version (works in the new version):
```
/fan-control.sh: line 32: echo: write error: Resource busy
```
perhaps a provision "if device is busy, do nothing" might fix that issue? @wolvi-lataniere

Reply:
Yes, I expected that to happen, that's why I wanted to test first.
I'll check how to mitigate that.